Frends.Community.Oracle/Frends.Community.Oracle.csproj

```diff
@@ -11,7 +11,7 @@
     <IncludeSource>true</IncludeSource>
```

Comment:
https://github.com/CommunityHiQ/Frends.Community.Oracle/blob/0ba68e08ec06cb30c969d4bf458decb06fd72dca/Frends.Community.Oracle/Frends.Community.Oracle.csproj#L31
You don't need to add this dependency, so it can be removed.

https://github.com/CommunityHiQ/Frends.Community.Oracle/blob/0ba68e08ec06cb30c969d4bf458decb06fd72dca/Frends.Community.Oracle/Frends.Community.Oracle.csproj#L32
You're not using this dependency, so it can be removed.